featuretools/tests/entityset_tests/test_es.py

```diff
@@ -694,12 +706,6 @@ def test_normalize_entity_new_time_index_error_check(es):
                             index='cancel_reason',
                             make_time_index='cancel_date')
 
-    es.normalize_entity(base_entity_id='customers',
```

Comment:
Can we add this back in to test the "not the original time index but still using additional variables" case